tasks/push_vm.yml

```diff
@@ -0,0 +1,50 @@
+- name: "Test if vm {{ vm.host }} @{{ vm.org }} does exist"
+  stat:
+    path: "{{ xen_vman_vm_config_path }}/{{ vm.org }}-{{ vm.host }}.cfg"
+  register: vm_config_file
+  changed_when: False
+
+- name: "Get old creation date from {{ vm.host }} @{{ vm.org }} (1/2)"
```

Comment:
Why do you fiddle with that date? It doesn't matter, just take the creation time of the inode

Reply:
Or leave it out, no one cares

Reply:
So no response here? Just a thumbs down?